### PR TITLE
EWPP-3320: Add new fields to Illustration lists paragraphs.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_flags.field_oe_subtitle.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - field.storage.paragraph.field_oe_subtitle
+    - paragraphs.paragraphs_type.oe_illustration_list_flags
+id: paragraph.oe_illustration_list_flags.field_oe_subtitle
+field_name: field_oe_subtitle
+entity_type: paragraph
+bundle: oe_illustration_list_flags
+label: Highlight
+description: ''
+required: false
+translatable: true
```
Comment (nagyad) [inline modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_flags.field_oe_subtitle.yml:14]: We should mark the field storage translatable only.

File: modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_icons.field_oe_center.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - field.storage.paragraph.field_oe_center
+    - paragraphs.paragraphs_type.oe_illustration_list_icons
+id: paragraph.oe_illustration_list_icons.field_oe_center
+field_name: field_oe_center
+entity_type: paragraph
+bundle: oe_illustration_list_icons
+label: 'Center the content'
+description: ''
+required: false
+translatable: true
```
Comment (nagyad) [inline modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_icons.field_oe_center.yml:14]: This should not be translatable.

File: modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_images.field_oe_size.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  module:
+    - options
+id: paragraph.oe_illustration_list_images.field_oe_size
+field_name: field_oe_size
+entity_type: paragraph
+bundle: oe_illustration_list_images
+label: Size
+description: ''
+required: true
+translatable: true
```
Comment (nagyad) [inline modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_images.field_oe_size.yml:16]: Do we need this to be translatable at all?

File: modules/oe_paragraphs_illustrations_lists/config/install/field.storage.paragraph.field_oe_size.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      value: medium
+      label: Medium
+    -
+      value: large
+      label: Large
+  allowed_values_function: ''
+module: options
+locked: false
+cardinality: 1
+translatable: true
```
Comment (nagyad) [inline modules/oe_paragraphs_illustrations_lists/config/install/field.storage.paragraph.field_oe_size.yml:26]: Do we need this translatable?

File: composer.json [hunk 5]
```diff
@@ -74,7 +75,8 @@
             }
         },
         "_readme": [
-            "Explicit requirement of symfony/phpunit-bridge to replace drupal/core-dev testing classes and traits."
+            "Explicit requirement of symfony/phpunit-bridge to replace drupal/core-dev testing classes and traits.",
+            "Using symfony/validator v6.2.5 for D10 like core-recommended does because 6.3 breaks BC with the signature changes to the ExecutionContextInterface."
         ]
```
Comment (nagyad) [inline composer.json:80]: We should update this to remind to remove once validator has a new release with the fix.

**Review by nagyad — CHANGES_REQUESTED** [review]

File: modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_flags.field_oe_subtitle.yml (hunk 1 above)
Comment (22Alexandra) [reply modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_flags.field_oe_subtitle.yml:14]: We have the other text fields translatable in this submodule and the rest.

File: modules/oe_paragraphs_illustrations_lists/config/install/core.entity_form_display.paragraph.oe_illustration_list_flags.default.yml [hunk 6]
```diff
@@ -15,20 +17,29 @@ targetEntityType: paragraph
 bundle: oe_illustration_list_flags
 mode: default
 content:
-  field_oe_illustration_columns:
-    weight: 2
-    settings: {  }
+  field_oe_center:
```
[the remark this answers is not in the document]
Comment (22Alexandra) [reply modules/oe_paragraphs_illustrations_lists/config/install/core.entity_form_display.paragraph.oe_illustration_list_flags.default.yml:20]: It follows the requirements: https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=EWPP&title=List+with+Illustrations+V2

File: modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_images.field_oe_subtitle.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    - field.storage.paragraph.field_oe_subtitle
+    - paragraphs.paragraphs_type.oe_illustration_list_images
+id: paragraph.oe_illustration_list_images.field_oe_subtitle
+field_name: field_oe_subtitle
+entity_type: paragraph
+bundle: oe_illustration_list_images
+label: Highlight
+description: ''
+required: false
+translatable: true
```
[the remark this answers is not in the document]
Comment (22Alexandra) [reply modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_images.field_oe_subtitle.yml:14]: It's a text field, the others are translatable too.

File: modules/oe_paragraphs_illustrations_lists/config/install/core.entity_form_display.paragraph.oe_illustration_list_flags.default.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (22Alexandra) [reply modules/oe_paragraphs_illustrations_lists/config/install/core.entity_form_display.paragraph.oe_illustration_list_flags.default.yml:20]: Indeed, that's fixed.

**Review by nagyad — DISMISSED** [review]

**Review by nagyad — APPROVED** [review]

**Review by nagyad — CHANGES_REQUESTED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by nagyad — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**